### DISCUSSION TITLE
Aggiornamento da core-2021.11.5 a core-2021.12.2

### DISCUSSION THREAD

**Discussion by Matteol82** [indomus/forum#1191] [HUB: Home Assistant]
Salve,
HASSIO mi sta dicendo che ha degli aggiornamenti di sistema da fare, più precisamente:

Home Assistant Core
Versione
core-2021.11.5
Ultima versione
core-2021.12.2

Operating System
Versione
6.6
Ultima versione
7.0

prima di effettuarli ho voluto controllare che tutto andasse bene attraverso l'addons Check Home Assistant configuration.
Questo però mi restituisce:
```
[19:51:43] ERROR: The configuration check did not pass!
[19:51:43] ERROR: See the output below for more details.
Testing configuration at /tmp/config
INFO:homeassistant.helpers.storage:Migrating core.device_registry storage from 1.1 to 1.2
INFO:homeassistant.helpers.storage:Migrating core.entity_registry storage from 1.1 to 1.4
Fatal error while loading config: function() argument 'code' must be code, not str
Failed config
  General Errors: 
    - function() argument 'code' must be code, not str

Successful config (partial)
[19:51:43] INFO: The full output has been written to /share/check_config.txt
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```

Da questo cosa dovrei capire?
Dov'è questa funzione con argomento 'code' non stringa?
Forse prima di verificare con  Check Home Assistant configuration devo comunque aggiornare l'Operating System?

Grazie.

**Comment by snakuzzo** [comment]
Leggi [QUI](https://community.home-assistant.io/t/cant-restart-home-assistant/366516/6)
